modulefiles/ufs_gaea-c5.intel.lua

```diff
@@ -0,0 +1,32 @@
+help([[
+  This module loads libraries required for building and running UFS Weather Model 
+  on the NOAA RDHPC machine Gaea C5 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea C5 ]===])
+
+load("PrgEnv-intel/8.3.3")
+load("intel-classic/2023.1.0")
+load("cray-mpich/8.1.25")
+load("python/3.9.12")
+
+prepend_path("MODULEPATH", "/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/modulefiles")
+
+stack_intel_ver=os.getenv("stack_intel_ver") or "2023.1.0"
+load(pathJoin("stack-intel", stack_intel_ver))
+
+stack_cray_mpich_ver=os.getenv("stack_cray_mpich_ver") or "8.1.25"
+load(pathJoin("stack-cray-mpich", stack_cray_mpich_ver))
+
+stack_python_ver=os.getenv("stack_python_ver") or "3.10.8"
+load(pathJoin("stack-python", stack_python_ver))
+
+load("ufs_common")
+
+unload("darshan-runtime")
+
```

Comment:
To run the executable in a workflow, I had to unload cray-libsci. Otherwise, there were two cray-libsci linked at runtime. The workflow included many other programs and libraries, including GSI, UFS_UTILS, the tracker, and others.

```suggestion
unload("cray-libsci")

```

Reply:
Thank you, @SamuelTrahanNOAA. The tests I conducted ran successfully without unloading the cray-libsci, but I rely on your experience that this is needed at other times.

Reply:
I've added a line for unload("cray-libsci") in ufs_gaea-c5.intel.lua.